src/cellfinder_core/detect/filters/volume/volume_filter.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         )
 
         self.cell_detector = get_cell_detector(
-            plane_shape=self.setup_params[0].shape,
+            plane_shape=self.setup_params[0].shape,  # type: ignore
```

Comment:
`mypy` doesn't know the dimensions of the array here, so gives `self.setup_params[0].shape` a type `Tuple[int, ...]` (no known length), but `plane_shape` expectes `Tuple[int, int` (length 2). I wasn't sure how to fix this, so ignored it for now.

Reply:
OK!

I'll open a low-priority issue for when we want to think more about types or want to refactor the code around `get_cell_detector` :shrug: